libnetwork/netutils/utils.go

```diff
@@ -124,6 +124,9 @@ func GenerateMACFromIP(ip net.IP) net.HardwareAddr {
 // GenerateRandomName returns a new name joined with a prefix.  This size
 // specified is used to truncate the randomly generated value
```

Comment:
The comment should be updated to describe the limitation on size, and why it is on place. However, it's unclear to me if this is the correct place to enforce the length, and whether a magic number should be hardcoded here, or if we should reference a constant.

In addition to updating the comment, could you elaborate on where this value is used and why it makes sense to enforce this invariant here? If it makes sense to enforce it here, does it make more sense to truncate?